### PR TITLE
Feature/add pages to storybook

### PR DESCRIPTION
**Pull request by raphaellueckl**
# Done is when (DoD):
- [x] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [x] The modified component properties have been added to the typescript typings.
- [x] Potential design changes have been reviewed by a designer.
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [x] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/pages/axa-ch-main/commercial-hero-banner.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <axa-commercial-hero-banner
+    variant="dark"
+    imagesource="https://brandcenter.axa.ch/m/4df4627fddd0534a/WIDE_1920_560-AXA_DE_DANNY_web.jpg"
+  >
+    <h2 slot="category">
+      Jetzt Haushalt und Privathaftpflicht versichern und profitieren
+    </h2>
+    <h1 slot="title">Bis zu CHF 200.– Rabatt</h1>
+    <axa-button-link
+      href="https://axa.ch"
```
Comment (danyball) [inline src/pages/axa-ch-main/commercial-hero-banner.js]: Is it possible to not give a link? Our showcase should not be left by clicking such a button.

Even the links in header and footer should'nt be removed in my opinion.

File: src/pages/axa-ch-main/story.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import { boolean, withKnobs } from '@storybook/addon-knobs';
+
+import Readme from '../../../README.md';
+import '../../components/10-atoms/text';
+import '../../components/10-atoms/heading';
+import commercialHeroBanner from './commercial-hero-banner';
+import footer from './footer';
+import policyFeatures from './policy-features';
+import navbar from './navbar';
+import footerSmall from './footer-small';
```
Comment (danyball) [inline src/pages/axa-ch-main/story.js:13]: You are wrapping our components to have a cleaner story.js?

File: src/pages/axa-ch-main/story.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+const story = storiesOf('Pages|AXA', module);
+story.addDecorator(withKnobs);
+story.addParameters({
+  readme: {
+    sidebar: Readme,
+  },
+});
+
+function getFirstChildOfEachAxaCustomElement() {
```
Comment (danyball) [inline src/pages/axa-ch-main/story.js:23]: :) nice

**Review by danyball — CHANGES_REQUESTED** [review]

**Review by LucaMele — DISMISSED** [review]

**Comment by LucaMele** [comment]
sehr nice

File: src/pages/axa-ch-main/commercial-hero-banner.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { html } from 'lit-html';
+
+export default html`
+  <axa-commercial-hero-banner
+    variant="dark"
+    imagesource="https://brandcenter.axa.ch/m/4df4627fddd0534a/WIDE_1920_560-AXA_DE_DANNY_web.jpg"
+  >
+    <h2 slot="category">
+      Jetzt Haushalt und Privathaftpflicht versichern und profitieren
```
Comment (01231) [inline src/pages/axa-ch-main/commercial-hero-banner.js]: do we want everything in german?

Comment (raphaellueckl) [reply src/pages/axa-ch-main/commercial-hero-banner.js]: No, nothing in german. I forgot to change that here.

**Review by 01231 — DISMISSED** [review]

File: src/pages/axa-ch-main/story.js (hunk 2 above)
Comment (raphaellueckl) [reply src/pages/axa-ch-main/story.js:13]: Yes, it help's with readabiliy and makes it reusable.

File: src/pages/axa-ch-main/commercial-hero-banner.js (hunk 1 above)
Comment (raphaellueckl) [reply src/pages/axa-ch-main/commercial-hero-banner.js]: Sure, I used # on other links, will do that here too, thanks!

File: src/pages/axa-ch-main/footer-small.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    >
+      DE
+    </a>
+    <a slot="language-item" href="https://axa.ch/fr/particuliers.html">
+      FR
+    </a>
+    <a slot="language-item" href="https://axa.ch/it/clienti-privati.html">
+      IT
+    </a>
+    <a slot="language-item" href="https://axa.ch/en/private-customers.html">
```
Comment (danyball) [inline src/pages/axa-ch-main/footer-small.js]: remove links....

File: src/pages/axa-ch-main/footer.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
+import { html } from 'lit-html';
+
+export default html`
+  <axa-footer>
+    <h2 slot="column-title">axa and you</h2>
+    <a
+      slot="column-item"
+      href="https://axa.ch/en/private-customers.html"
```
Comment (danyball) [inline src/pages/axa-ch-main/footer.js]: remove links

**Review by danyball — CHANGES_REQUESTED** [review]

**Review by danyball — APPROVED** [review]

**Pull request merged**